chrome/content/zotero/bibliography.js

```diff
@@ -34,9 +34,13 @@
 
 var Zotero_File_Interface_Bibliography = new function() {
 	var _io, _saveStyle;
+	var selectedLocale = "";
+	var defaultStyleLocale = "";
 	
 	this.init = init;
 	this.styleChanged = styleChanged;
+	this.populateLocaleList = populateLocaleList;
+	this.localeChanged = localeChanged;
```

Comment:
We don't use this style anymore, since it's error-prone. Should just use ``this.populateLocaleList = function () {`` below.